seahorn/jobs2/ring_buffer_acquire2/aws_ring_buffer_acquire_harness2.c

```diff
@@ -67,11 +74,15 @@ int main(void) {
     sassert(!(is_front_valid_state(&ring_buf_old) &&
               is_middle_valid_state(&ring_buf)));
   } else {
-    sassert(ring_buffers_are_equal(&ring_buf, &ring_buf_old));
+    sassert(!sea_is_modified((char *)ring_buf.allocator) &&
```

Comment:
Can you provide some intuition for why `ring_buffers_are_equal` and this `sassert` are equivalent?

To me it seems like we want to check that `allocation` is unmodified and the rest of the equality checks remain.